File: Data Structures & Algorithms/duplicate-integer/submission-0.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
      List<int> seenList = new List<int>();

      for (int i = 0; i < nums.Length; i++) {
        if (seenList.Contains(nums[i])) {
            return true;
        } else {
            seenList.Add(nums[i]);
        }
      }

      return false;
    }
}

File: Data Structures & Algorithms/non-cyclical-number/submission-0.py
class Solution:
    def isHappy(self, n: int) -> bool:
        done = False
        res = n
        seenList = []

        while not done:
            l = str(res)
            res = 0

            for i in range(len(l)):
                digit = int(l[len(l) - 1 - i])
                res += digit * digit
            
            print(seenList)
            
            if res == 1:
                return True
            elif res in seenList:
                return False
            
            seenList.append(res)